### ISSUE TITLE
[RFE] Include simple-scan in default install

### ISSUE THREAD

**Issue by mo-pyy** [fedora-silverblue/issue-tracker#242]
**Is your feature request related to a problem? Please describe.**
When I want to scan from a local usb printer, i need to layer the simple-scan package.

**Describe the solution you'd like**
I think the simple-scan package should be included in the default installation,
because scanning is a very common operation.

**Describe alternatives you've considered**
I have installed simple-scan in a toolbox container, but in the container, only network scanning works. A flatpak doesn't seem to be available.

**Comment by travier** [comment]
Silverblue includes by default the same applications that are included in Workstation. Is simple-scan included in Workstation? If it is and we can make a Flatpak for it then, we can include it. Otherwise, you would have to ask the Workstation group to include it first.

**Comment by tpopela** [comment]
With SImple Scan it will be a little bit harder - it will need to be put into the base image as we don't have the Flatpak device portal yet :/ - see https://github.com/flatpak/xdg-desktop-portal/issues/218 and https://gitlab.gnome.org/GNOME/simple-scan/-/issues/21

**Comment by kowalski7cc** [comment]
@travier Yes, Workstation 36 does include simple-scan as "Document Scanner".
@tpopela For now it could be included in the base image like on Workstation, or there is a flatpak https://gitlab.gnome.org/GNOME/simple-scan/-/blob/master/build-aux/flatpak/org.gnome.SimpleScan.json. Even if it doesn't provide good sandboxing, should be still better than no scanner at all...

**Comment by LorbusChris** [comment]
+1 to including this - the flatpak doesn't seem to be ready, nor does it have proper sandboxing (as the device portal is still missing).

**Comment by travier** [comment]
> Simple Scan is really needed for me to adopt Silverblue. Hopefully next time. For now it will be regular Fedora.

Please remember that you can overlay packages on Silverblue and it is perfectly fine and supported. overlaying Simple Scan until it is provided by default / packaged as a Flatpak is the current workaround.

**Comment by travier** [comment]
Simple Scan is now available as a Flatpak in Fedora so we can add it for Fedora 40.

**Comment by travier** [comment]
PR: https://pagure.io/pungi-fedora/pull-request/1229

**Comment by LorbusChris** [comment]
The Fedora Flatpak works for me with an IPP scanner (advertised through mDNS)

**Comment by miabbott** [comment]
Installed the Flatpak and was able to discover my HP printer/scanner on my wireless network; scanned a document successfully.

**Comment by tpopela** [comment]
I've tried it with my old all-in-one HP printer and Fedora Flatpak as expected doesn't see the scanner connected through USB (it works if simple-scan is layered).

**Comment by travier** [comment]
OK, so this works for network'ed scanners but not for USB scanners. What should we do? Include it as Flatpak nonetheless or include it in the base image given that it's the only way it will work? Or we keep things as is.

**Comment by kowalski7cc** [comment]
I think that including it in the base install until flatpak gets support for a future USB portal is probably the best solution from a user perspective, if the size doesn't grow too much.
The issue was opened when the simple scan flatpak didn't exist so it was an issue on silverblue, now that it's partially resolved.

**Comment by tpopela** [comment]
I would prefer to main the status quo (who needs it will layer it) as there's some work happening on the USB portal. Also I don't want to put additional burden on upstream/Fedora maintainers when they will get reports that what's preinstalled in Silverblue doesn't work for them (without any visible hint what to do).

**Comment by LorbusChris** [comment]
I'm fine with the status quo, too.

@miabbott I'm wondering, did you have to enable mDNS resolution for that, or did it work out of the box with Fedora's stock config?
